src/EnergyPlus/ZoneAirLoopEquipmentManager.cc

```diff
@@ -814,28 +814,18 @@ namespace ZoneAirLoopEquipmentManager {
             }
         }
         if (ProvideSysOutput) {
+            int OutletNodeNum = AirDistUnit(AirDistUnitNum).OutletNodeNum;
+            int ZoneAirNode = ZoneEquipConfig(ControlledZoneNum).ZoneNode;
+            SpecHumOut = Node(OutletNodeNum).HumRat;
+            SpecHumIn = Node(ZoneAirNode).HumRat;
             // Sign convention: SysOutputProvided <0 Zone is cooled
             //                  SysOutputProvided >0 Zone is heated
-            SpecHumOut = Node(AirDistUnit(AirDistUnitNum).OutletNodeNum).HumRat;
-            SpecHumIn = Node(ZoneEquipConfig(ControlledZoneNum).ZoneNode).HumRat;
-            if (AirDistUnit(AirDistUnitNum).EquipType_Num(1) == SingleDuctConstVolNoReheat) {
-                // Use old direct air method to avoid diffs for now
-                SysOutputProvided = Node(AirDistUnit(AirDistUnitNum).OutletNodeNum).MassFlowRate *
-                                    (Psychrometrics::PsyHFnTdbW(Node(AirDistUnit(AirDistUnitNum).OutletNodeNum).Temp,
-                                                                Node(ZoneEquipConfig(ControlledZoneNum).ZoneNode).HumRat) -
-                                     Psychrometrics::PsyHFnTdbW(Node(ZoneEquipConfig(ControlledZoneNum).ZoneNode).Temp,
-                                                                Node(ZoneEquipConfig(ControlledZoneNum).ZoneNode).HumRat));
-            } else {
-                Real64 CpAirAvg = PsyCpAirFnW(0.5 * (SpecHumOut + SpecHumOut));
-                SysOutputProvided = Node(AirDistUnit(AirDistUnitNum).OutletNodeNum).MassFlowRate * CpAirAvg *
-                                    (Node(AirDistUnit(AirDistUnitNum).OutletNodeNum).Temp - Node(ZoneEquipConfig(ControlledZoneNum).ZoneNode).Temp);
-            }
-
+            SysOutputProvided = Node(OutletNodeNum).MassFlowRate * Psychrometrics::PsyDeltaHSenFnTdb2W2Tdb1W1(Node(OutletNodeNum).Temp, SpecHumOut, Node(ZoneAirNode).Temp, SpecHumIn); // sensible {W};
+            ;
```

Comment:
The new method (function) uses average humidity ratio for *Cp* air calculation for all air terminal types whereas the old method was using zone air humidity ratio for constant volume no reheat air terminal and zone entering air humidity ratio for all other air terminal types.